### DISCUSSION TITLE
Regarding the soil moisture output obtained form mHM

### DISCUSSION THREAD

**Discussion by bishal-bi** [mhm-ufz/mHM#109] [Q&A]
I have used the ERA5-Land soil moisture satellite products (monthly) to validate the soil water content (SWC) obtained from the mHM. ERA5-Land provides volumetric water content (VWC) in fraction across various depth, which is suitable for validating soil moisture value obtained from mHM across varying depths. The SWC obtained from the mHM when divided with their respective depth, provides value similar to VWC, changing them into comparable units.
Simulation data				 Satellite data 
swc_L01                       (0-50)             --->> swvL1 (0-70)              lvl 1
swc_L02 + swc_L03   (50-300)         --->> swvL2 (70-280)             lvl 2
swc_L04 + swc_L05   (300-1000)    --->> swvL3 (280-1000)          lvl 3
depths are in mm

The grid pattern point randomization was used and soil moisture was compared.

![image](https://github.com/user-attachments/assets/a0a2c01a-5a53-4ad4-9d00-eb87c1bc6c3f)

From the time-series graph plotted and the correlation study, we concluded that soil moisture has been accurately simulated up to a depth of 300 mm.
Queries:
1. While observing average monthly soil moisture values at various level, SWC_L05 and SWC_L06 seems to have nearly constant value throughout the year. In our study region, 70-80% of the precipitation occurs from June to September (within four months) and ERA5_Land satellite data also shows variability of SM around the year. Thus the output is not reasonable. We made similar observations at some points on SWC_L04 too. What may be the reason for such output?
2. The poorest correlation exists, near the large water body, for point 1 (with lvl 1 value of 0.72), as the water body does not lie in our domain, and might effect the satellite average SM value. For the points at Himalayan region, the average correlation at level 1 is 0.828 (with the lowest 0.756 for P3) and further decreases for lvl 2. Similarly, for the hilly and terai region, the average correlation at level 1 is 0.917 (with lowest 0.836 for P12), and the correlation increases for lvl 2. This shows relatively poor performance in mountainous region. Is it due to poor snow handling of mHM? Please pass the comments.
3. Would you like to make some suggestions regarding validating mHM outputs?

Thank you for your time and help!!

Additional info:
<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 15">
<meta name=Originator content="Microsoft Word 15">
<link rel=File-List
href="file:///C:/Users/dell/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">

<link rel=themeData
href="file:///C:/Users/dell/AppData/Local/Temp/msohtmlclip1/01/clip_themedata.thmx">
<link rel=colorSchemeMapping
href="file:///C:/Users/dell/AppData/Local/Temp/msohtmlclip1/01/clip_colorschememapping.xml">

<style>

</style>

</head>

<body lang=EN-US style='tab-interval:.5in'>

Point | Lvl 1 Correlation | Lvl 1 RMSE | Lvl 2 Correlation | Lvl 2 RMSE | Lvl 3 Correlation | Lvl 3 RMSE
-- | -- | -- | -- | -- | -- | --
P1 | 0.72810 | 0.21239 | 0.77934 | 0.14746 | nan | nan
P2 | 0.88835 | 0.12225 | 0.67654 | 0.02851 | nan | nan
P3 | 0.75678 | 0.08244 | 0.83043 | 0.05243 | nan | nan
P4 | 0.91143 | 0.08851 | 0.93678 | 0.05637 | 0.87042 | 0.04653
P5 | 0.83143 | 0.08766 | 0.76825 | 0.05095 | 0.73678 | 0.03998
P6 | 0.82457 | 0.07682 | 0.79907 | 0.04176 | 0.70777 | 0.03701
P7 | 0.83361 | 0.07607 | 0.83546 | 0.08468 | 0.75196 | 0.09805
P8 | 0.80892 | 0.06623 | 0.81979 | 0.03761 | 0.70487 | 0.03685
P9 | 0.90532 | 0.13434 | 0.91353 | 0.11710 | 0.80570 | 0.09007
P10 | 0.96168 | 0.02648 | 0.96832 | 0.04040 | 0.84764 | 0.05903
P11 | 0.91544 | 0.05142 | 0.97193 | 0.02600 | 0.89280 | 0.04985
P12 | 0.83640 | 0.07416 | 0.75836 | 0.03685 | 0.75419 | 0.04986
P13 | 0.84056 | 0.09607 | 0.80755 | 0.04056 | 0.74945 | 0.04256
P14 | 0.82270 | 0.07383 | 0.83828 | 0.06390 | 0.76242 | 0.04687
P15 | 0.78141 | 0.07959 | 0.81424 | 0.04292 | 0.63639 | 0.04654
P16 | 0.95912 | 0.05620 | 0.99165 | 0.03161 | 0.85341 | 0.09265
P17 | 0.93026 | 0.08041 | 0.98146 | 0.06206 | 0.88606 | 0.06920
P18 | 0.91490 | 0.04709 | 0.98695 | 0.02120 | 0.90820 | 0.05425
P19 | 0.90750 | 0.04761 | 0.98421 | 0.02064 | 0.92477 | 0.05745
P20 | 0.92930 | 0.06786 | 0.94680 | 0.04040 | 0.85657 | 0.04959
P21 | 0.89284 | 0.06170 | 0.85297 | 0.03210 | 0.81785 | 0.04424

</body>

</html>

**Comment by StephanThober** [comment]
Hi Bishal,

this is a very interesting analysis. Please note that ERA5-Land is not a satellite-based product, but a reanalysis. It is created by assimilating observations into the weather forecasting model. Regarding your points:

1. Is this observed across all points? Even in L3, some of the correlations you report are quite high. Have you calibrated the model and if so, which parameters did you choose for calibration?
2. Have you used ERA5-Land as forcing dataset for mHM?
3. Validating mHM (in fact that is true for any model) output for more than one variable is always advisable. So looking at soil moisture as you do here is very good, but you should also check river discharge, evapotranspiration, and in your case, also snow pack would be very interesting.

Best,
Stephan

**Reply by bishal-bi** [reply]
Thank you sir,
I completely agree with your points on ERA5-Land. ERA5-Land has only been used for validating soil moisture (SM). The results obtained are quite exciting, as they nearly match each other, particularly for snow-free regions. I have tried to clarify my queries with the following points:
1. Yes, similar observations are made at all points. I will ensure the summarized observation table is attached at the end. Additionally, it is noted that SWC_L04 exhibits low variation throughout the year in mountainous regions.
![image](https://github.com/user-attachments/assets/bbf3ec96-94ea-4ca4-b05f-f0df3765036d)
The graphical representation is of L3-P11. The high correlation value for some points in L3 is attributed to seasonal synchronization, as similar increasing and decreasing trends are observed, despite differences in extent. Additionally, the mean of both datasets lie within a similar range. For the lower correlation in Lvl 1 and Lvl 2, the extent of both datasets is within a similar range, thus we considered better SM for lvl 1 and lvl 2 than Lvl 3 despite of low correlation.
![image](https://github.com/user-attachments/assets/7fb8a8f1-efcc-4dc7-8e9f-fd0005e040e9)
The graphical representation is of L1-P3 (least correlated among L1).
Yes, the model parameters were optimized. All global parameters were optimized except those not recommended to be optimized by mHM documentation.

2. No, ERA5-Land has only been used for validation. I have used dataset from mHM global database, provided by Olda, as forcing dataset for mHM.

3. Thank you for your suggestions. I will work on it.

Summarized table:
Table: Summary on soil moisture variation for the year 2010 of study domain (all values are in mm)
Max-Min represents difference in Maximum SM and Minimun SM (VWC in this case) for the year 2010.
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/dell/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/dell/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">


  | SWC_L01 (0-50) |   | SWC_L02 (50-150) |   | SWC_L03 (150-300) |   | SWC_L04 (300-600) |   | SWC_L05   (600-1000) |   | SWC_L06   (1000-2000) |  
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
  | Max-Min | Mean | Max-Min | Mean | Max-Min | Mean | Max-Min | Mean | Max-Min | Mean | Max-Min | Mean
P1 | 12.406 | 12.596 | 20.422 | 29.242 | 7.221 | 52.795 | 2.419 | 102.3 | 0.109 | 137.433 | 0.002 | 342.994
P2 | 12.752 | 14.553 | 13.349 | 35.905 | 4.56 | 59.972 | 0.979 | 105.491 | 0.04 | 139.881 | 0.001 | 348.417
P3 | 13.802 | 13.3 | 19.648 | 32.005 | 5.51 | 57.697 | 1.101 | 107.923 | 0.042 | 143.017 | 0.001 | 356.37
P4 | 13.516 | 11.675 | 26.903 | 23.719 | 38.118 | 44.196 | 18.982 | 93.907 | 0.825 | 133.109 | 0.012 | 334.459
P5 | 15.979 | 12.604 | 30.179 | 25.494 | 35.58 | 50.907 | 10.29 | 100.527 | 0.474 | 138.278 | 0.008 | 345.44
P6 | 14.635 | 12.963 | 30.88 | 25.853 | 36.646 | 50.934 | 11.201 | 99.398 | 0.515 | 137.09 | 0.009 | 342.88
P7 | 14.035 | 13.647 | 17.507 | 33.021 | 5.899 | 59.452 | 1.373 | 105.326 | 0.059 | 139.946 | 0.001 | 348.755
P8 | 13.859 | 10.674 | 22.355 | 26.699 | 7.61 | 52.598 | 2.815 | 103.455 | 0.145 | 139.598 | 0.003 | 348.428
P9 | 12.339 | 12.248 | 25.064 | 25.134 | 36.955 | 42.705 | 29.981 | 88.185 | 1.518 | 132.741 | 0.023 | 339.307
P10 | 12.993 | 14.3 | 25.899 | 29.597 | 30.61 | 51.51 | 12.656 | 92.362 | 0.687 | 131.501 | 0.013 | 335.053
P11 | 12.952 | 12.522 | 25.885 | 25.792 | 37.897 | 44.986 | 27.159 | 88.667 | 1.297 | 131.806 | 0.02 | 336.589
P12 | 14.016 | 14.897 | 25.111 | 30.776 | 19.554 | 56.067 | 5.514 | 99.407 | 0.278 | 135.184 | 0.005 | 337.807
P13 | 14.917 | 14.367 | 25.805 | 29.238 | 15.413 | 55.454 | 4.297 | 101.22 | 0.214 | 137.575 | 0.004 | 344.146
P14 | 14.957 | 10.666 | 24.732 | 24.538 | 8.867 | 53.069 | 1.979 | 105.011 | 0.079 | 136.608 | 0.001 | 336.788
P15 | 13.077 | 10.758 | 19.031 | 27.276 | 10.264 | 52.044 | 2.212 | 99.732 | 0.085 | 132.976 | 0.001 | 331.917
P16 | 9.89 | 9.9 | 20.824 | 19.977 | 31.911 | 32.519 | 37.989 | 85.877 | 1.633 | 135.077 | 0.02 | 341.665
P17 | 8.172 | 10.2 | 17.618 | 20.647 | 27.713 | 32.867 | 35.224 | 79.567 | 1.66 | 124.319 | 0.021 | 316.081
P18 | 11.82 | 11.936 | 23.845 | 23.803 | 35.67 | 40.407 | 33.738 | 86.603 | 1.686 | 132.492 | 0.024 | 337.852
P19 | 11.265 | 12.339 | 22.932 | 24.326 | 33.689 | 40.8 | 29.322 | 88.566 | 1.323 | 132.71 | 0.019 | 338.163
P20 | 13.959 | 13.665 | 28.309 | 27.169 | 40.411 | 47.642 | 25.338 | 90.017 | 1.237 | 132.92 | 0.021 | 338.735
P21 | 14.299 | 13.737 | 27.845 | 27.397 | 31.009 | 50.301 | 8.727 | 101.117 | 0.377 | 137.798 | 0.006 | 343.789



</body>

</html>

**Reply by StephanThober** [reply]
Thank you for sharing a deeper analysis. For P17, this is a nice example that soil moisture is a highly model dependent quantity. Do you actually think that it is realistic that the soil dries out to values of less than 0.2 of vwc? It seems very low to me.

Regarding P3, the differences during winter for both models might actually be related to the occurrence of snow pack.

**Reply by bishal-bi** [reply]
Thank you for the response. The VWC value at its driest state also depends on soil type. Soil type may have led to this low VWC. In any case, lvl1 and lvl2 correlation is really good and the lowest value goes below 0.20 for both satellite data and the modeling which is presented as:
![image](https://github.com/user-attachments/assets/0c2e7b97-4c76-4227-8a65-2b40be36b971)
![image](https://github.com/user-attachments/assets/8beb74fb-afd2-4f56-a015-4c5c9b297016)
For this reason, i agreed with the low value of vwc.

In this analysis, my primary concern is on no change of vwc for the depths greater than 60 cm for all regions. I am trying to make a study on forest fires. The modeling output shows that the root zone of the trees remains in constant moisture throughout the year. (For P17, lvl1 has tentative lowest value 0.16, lvl 2 has 0.18 but lvl 3 has lowest value of 0.27, nearly constant throughout the year).

Any snow cover data products that you would like to recommend for validation?

**Reply by diegodinamarca** [reply]
Hi Bishal, I'm just going to share some insights. We have been working with the soil moisture (SM) output too and we have observed the same problem of constant soil moisture but at greater depths (we are modelling up to 2 to 4 m of soil depth) in a mediterranean zone without snow cover. We have tested different soil maps to study their influence on the modeled soil moisture and so far we have seen that using different maps results in non constant deep SM. Nevertheless, the SM seems to be heavily influenced by the global parameters of the model, specially at greater depths, so we are testing the sensitivity of the model by modelling the SM using several calibrated parameter sets.

**Reply by StephanThober** [reply]
Hi Bishal,
I agree that this behaviour might be related with soil parameters. Could you share the global parameter set that you obtained after calibration.

For snow validation datasets, you can have a look at this paper: https://tc.copernicus.org/articles/13/2221/2019/

Best,
Stephan

**Reply by bishal-bi** [reply]
Here is the global parameter set obtained after calibration.
! This file was automatically generated from file containing multiple parameter sets:
!     /home/bibek/mhm_workshop/mhm_installation_work/mhm/pre-proc/parameter_set_best.txt
! This is set #1
!
!global_parameters
!PARAMETER                       lower_bound  upper_bound          value   FLAG  SCALING
!interception
&interception1
canopyInterceptionFactor           =  0.1500,      0.4000,     	0.206856459	,     1,       1
/		
		
! snow		
&snow1		
snowTreshholdTemperature           = -2.0000,      2.0000,     	-0.82954800635215964	,     1,       1
degreeDayFactor_forest             =  0.0001,      4.0000,     	2.5458037384878587	,     1,       1
degreeDayFactor_impervious         =  0.0000,      1.0000,     	0.98953331503461772 	,     1,       1
degreeDayFactor_pervious           =  0.0000,      2.0000,     	1.9287103251161044	,     1,       1
increaseDegreeDayFactorByPrecip    =  0.1000,      0.9000,     	0.36189755884391128	,     1,       1
maxDegreeDayFactor_forest          =  0.0000,      8.0000,     	 6.9585272491693342	,     1,       1
maxDegreeDayFactor_impervious      =  0.0000,      8.0000,     	1.4149040666354038	,     1,       1
maxDegreeDayFactor_pervious        =  0.0000,      8.0000,     	4.9596194891417333	,     1,       1
/		
		
! soilmoisture		
&soilmoisture1		
orgMatterContent_forest            =  0.0000,      20.000,    	9.3492258569289532	,     1,       1
orgMatterContent_impervious        =  0.0000,      1.0000,    	0.51847580089944323	,     1,       1
orgMatterContent_pervious          =  0.0000,      4.0000,    	1.8779914690605053	,     1,       1
PTF_lower66_5_constant             =  0.6462,      0.9506,    	0.79575768136604674	,     1,       1
PTF_lower66_5_clay                 =  0.0001,      0.0029,    	6.6199759232371579E-004	,     1,       1
PTF_lower66_5_Db                   = -0.3727,     -0.1871,    	-0.26002786768711011	,     1,       1
PTF_higher66_5_constant            =  0.5358,      1.1232,    	0.70357088475828855	,     1,       1
PTF_higher66_5_clay                = -0.0055,      0.0049,    	-4.6260024136016587E-00	,     1,       1
PTF_higher66_5_Db                  = -0.5513,     -0.0913,    	-0.19998148488601947	,     1,       1
PTF_Ks_constant                    = -1.2000,     -0.2850,    	-0.94810483233546172	,     1,       1
PTF_Ks_sand                        =  0.0060,      0.0260,    	1.4681833812253521E-002	,     1,       1
PTF_Ks_clay                        =  0.0030,      0.0130,    	6.9597951962294645E-003	,     1,       1
PTF_Ks_curveSlope                  =  1.0000,     150.000,    	60.960000000000001	,     1,       1
rootFractionCoefficient_forest     =  0.9000,      0.9990,    	0.90085882943459494	,     1,       1
rootFractionCoefficient_impervious =  0.9000,      0.9500,    	0.90323845591661700	,     1,       1
rootFractionCoefficient_pervious   =  0.0010,      0.0900,    	2.9240707133443480E-002	,     1,       1
infiltrationShapeFactor            =  1.0000,      4.0000,    	2.4064767085204624	,     1,       1
/		
		
! directSealedAreaRunoff		
&directRunoff1		
imperviousStorageCapacity          =  0.0000,      5.0000,     	3.0500465095118310	,     1,       1
/		
		
! meteoCorrectionFactor		
&actualET1		
PET_a_forest =      0.300000000000,       1.300000000000,     	0.80880245251721883	,     1,       1
PET_a_impervious =      0.300000000000,       1.300000000000,     	0.61788783243750456	,     1,       1
PET_a_pervious =      0.300000000000,       1.300000000000,     	1.2921537312161480	,     1,       1
PET_b =      0.000000000000,       1.500000000000,       	0.47526526727380564	,     1,       1
PET_c =     -2.000000000000,       0.000000000000,       	-0.10997334512681714	,     1,       1
/		
		
! interflow		
&interflow1		
interflowStorageCapacityFactor     =  75.000,      200.00,     	91.579553838688767	,     1,       1
interflowRecession_slope           =  0.0000,      10.000,     	9.8170642880463355	,     1,       1
fastInterflowRecession_forest      =  1.0000,      3.0000,     	2.9262367954927022	,     1,       1
slowInterflowRecession_Ks          =  1.0000,      30.000,     	20.626909578608231	,     1,       1
exponentSlowInterflow              =  0.0500,      0.3000,     	5.1221116897139742E-002	,     1,       1
/		
		
		
! percolation		
&percolation1		
rechargeCoefficient                =  0.0000,      50.000,     	46.640554851229851	,     1,       1
rechargeFactor_karstic             = -5.0000,      5.0000,     	2.3479749103505139	,     1,       1
gain_loss_GWreservoir_karstic      =  1.0000,      1.0000,     	1.0000000000000000	,     0,       1
/		
		
! routing		
&routing3		
slope_factor =      0.100000000000,     100.000000000000,     	30	,     0,       1
/		
		
! geological parameters (ordering according to file "geology_classdefinition.txt")		
! this parameters are NOT REGIONALIZED yet, i.e. these are <beta> and not <gamma>		
&geoparameter		
GeoParam(1,:)                      =  1.0000,      1000.00,     	100.00000000000000 "	,     1,       1
GeoParam(2,:)                      =  1.0000,      1000.00,     	100.00000000000000	,     1,       1
GeoParam(3,:)                      =  1.0000,      1000.00,     	26.290726983125069	,     1,       1
GeoParam(4,:)                      =  1.0000,      1000.00,     	126.75664415368759	,     1,       1
GeoParam(5,:)                      =  1.0000,      1000.00,     	512.66611031398338	,     1,       1
GeoParam(6,:)                      =  1.0000,      1000.00,     	203.51986667163163	,     1,       1
GeoParam(7,:)                      =  1.0000,      1000.00,     	62.649892841155463	,     1,       1
GeoParam(8,:)                      =  1.0000,      1000.00,     	100.00000000000000	,     1,       1
GeoParam(9,:)                      =  1.0000,      1000.00,     	1.8020817645351810	,     1,       1
GeoParam(10,:)                      =  1.0000,      1000.00,     	41.460726067542282	,     1,       1
GeoParam(11,:)                      =  1.0000,      1000.00,     	100.00000000000000	,     1,       1
GeoParam(12,:)                      =  1.0000,      1000.00,     	193.59472917219190	,     1,       1
GeoParam(13,:)                      =  1.0000,      1000.00,     	273.72298196934202	,     1,       1
GeoParam(14,:)                      =  1.0000,      1000.00,     	100.00000000000000	,     1,       1
GeoParam(15,:)                      =  1.0000,      1000.00,     	100.00000000000000	,     1,       1
GeoParam(16,:)                      =  1.0000,      1000.00,     	100.00000000000000
	,     1,       1
/
